code/modules/mob/living/carbon/human/species_types/abductors.dm

```diff
@@ -31,8 +33,9 @@
 
 
 /datum/species/abductor/get_physical_attributes()
+	//bubber edit - Abductor Nerf
 	return "Abductors do not need to breathe, eat, do not have blood, a heart, stomach, or lungs and cannot be infected by human viruses. \
-		Their hardy physique prevents their skin from being wounded or dismembered, but their chunky tridactyl hands make it hard to operate human equipment."
+	Their chunky tridactyl hands make it hard to operate human equipment."
```

Comment:
Override this modularly by redefining /datum/species/abductor/get_physical_attributes() in modular files